src/foam/nanos/crunch/ui/UserCapabilityJunctionWAO.js

```diff
@@ -69,7 +69,8 @@ foam.CLASS({
       ) : this.crunchService.updateJunction(
         null, wizardlet.capability.id, null, null
       );
-      return p.then(ucj => { return ucj; });
+      return p.then(ucj => { return ucj; })
```

Comment:
Do you need then then()? Will just the .catch do?

Reply:
Yes just the catch is fine. This promise resolves after a failure.